sftp/sftp.go

```diff
@@ -508,9 +508,9 @@ func (o *Object) Hash(r fs.HashType) (string, error) {
 	err = fs.ErrHashUnsupported
 	var outputBytes []byte
 	if r == fs.HashMD5 {
-		outputBytes, err = session.Output("md5sum " + o.path() + " | awk '{ print $1 }'")
+		outputBytes, err = session.Output("md5sum \"" + o.path() + "\" | awk '{ print $1 }'")
```

Comment:
I guess in addition to using `quotemeta`? Looks like that one doesn't escape spaces.